test/go-tests/test_importer_test.go

```diff
@@ -0,0 +1,163 @@
+	require.NoError(t, err)
+	require.NotContains(t, approvalCLIOutput, fmt.Sprintf("No services %s found in project", serviceName))
+
+	// Check if secret exists
+	err = checkIfSecretExists("slack-webhook")
+	require.NoError(t, err)
+
+	// Validate webhook
+	exists, err := CheckIfWebhookSubscriptionExists(projectName, "sh.keptn.event.evaluation.triggered")
+	require.NoError(t, err)
+	require.True(t, exists, "Webhook service subscription does not exist")
+
+}
+
+func Test_ImportCorrectManifestNonExistingProject(t *testing.T) {
+	projectName := "keptn-importer-test-non-existing"
+	wrongProjectName := "ketpn-importer-test-non-existing"
+	errorMessage := "project not found"
+	expectedErrorCode := 424
+
+	t.Logf("Creating a new project %s with Gitea Upstream", projectName)
+	shipyardFilePath, err := CreateTmpShipyardFile(importerShipyard)
+	require.Nil(t, err)
+	projectName, err = CreateProject(projectName, shipyardFilePath)
+	require.Nil(t, err)
+
+	// Convert folder to Zip
+	err = createZipFileFromDirectory("../assets/import/sample-package/", "./sample-package.zip", false)
+	require.Nil(t, err)
+
+	// Getting Keptn API credentials
+	_, keptnApiUrl, err := GetApiCredentials()
+	require.Nil(t, err)
+
+	// Make API call with ZIP file
+	responseCode, err := ImportUploadZipToProject(fmt.Sprintf("%s/v1/import", keptnApiUrl), wrongProjectName, "./sample-package.zip")
+	require.Equal(t, expectedErrorCode, responseCode, fmt.Sprintf("Expected response status %d but got %d", expectedErrorCode, responseCode))
+	require.ErrorContains(t, err, errorMessage, fmt.Sprintf("Could not find expected error message: %s", errorMessage))
+	require.Error(t, err)
+}
+
+func Test_ImportMalformedZipFileCorrectName(t *testing.T) {
+	projectName := "keptn-importer-test-malformed-zip"
+	errorMessage := "Error opening import archive"
+	expectedErrorCode := 415
+
+	t.Logf("Creating a new project %s with Gitea Upstream", projectName)
+	shipyardFilePath, err := CreateTmpShipyardFile(importerShipyard)
+	require.Nil(t, err)
+	projectName, err = CreateProject(projectName, shipyardFilePath)
+	require.Nil(t, err)
+
+	// Convert folder to Zip
+	err = createZipFileFromDirectory("../assets/import/invalid-package/", "./invalid-package.zip", false)
+	require.Nil(t, err)
+
+	// Getting Keptn API credentials
+	_, keptnApiUrl, err := GetApiCredentials()
+	require.Nil(t, err)
+
+	// Make API call with ZIP file
+	responseCode, err := ImportUploadZipToProject(fmt.Sprintf("%s/v1/import", keptnApiUrl), projectName, "./invalid-package.zip")
+	require.Equal(t, expectedErrorCode, responseCode, fmt.Sprintf("Expected response status %d but got %d", expectedErrorCode, responseCode))
+	require.ErrorContains(t, err, errorMessage, fmt.Sprintf("Could not find expected error message: %s", errorMessage))
+	require.Error(t, err)
+}
+
+func ImportUploadZipToProject(urlPath, projectName, filePath string) (int, error) {
```

Comment:
Not sure what's the best practice in Keptn go-tests, but I would move 
```
_, keptnApiUrl, err := GetApiCredentials()
```
into the `ImportUploadZipToProject` function, and avoid code duplication in all three tests.

Reply:
We also already use `GetApiCredentials()` in the `ImportUploadZipToProject` function so it definitely makes sense.